src/modules/formBuilder/components/FormBuilderPage.tsx

```diff
@@ -44,6 +44,9 @@ const FormBuilderPage = () => {
 
   const [errorState, setErrorState] = useState<ErrorState>(emptyErrorState);
 
+  useEffect(() => {
+    setErrorState(emptyErrorState);
+  }, [selectedItem, setErrorState]);
```

Comment:
Fixes a bug where if there was an error in the item editor and you close the editor, the error still appears on the main page.

Reply:
Thanks 🙏  this has been bugging me too!